src/Microsoft.AspNetCore.Buffering/baseline.netcore.json

```diff
@@ -0,0 +1,66 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.Buffering, Version=2.0.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
?? This is a prototype, there is no version 2.0.2.

Reply:
Hmm yes that's an odd version #...

Reply:
As in, it should be 0.4.0 or similar...

Reply:
Can we remove baselines for prototypes?

Reply:
@Tratcher yeah good point I think that is the right thing to do. There can't be a breaking change in a pre-1.0.0 package.